bolt-design-system/bolt
# Example integrations auto dependencies update

## Jira

http://vjira2:8080/browse/BDS-1087

## Summary

The example integrations we have in our repo (ie our Drupal Lab and Vue.js examples) aren’t getting as well maintained as the components themselves.

## Details

Create a node script to update dependencies in example integrations. This script can be used to update other bolt dependencies in a different folder by passing paths to the array. 

## How to test

Run this code locally change dependencies in `package.json` on some bolt components in the `example-integrations` folder eg. in `vue` or `angular` folder. Then run `npm run dependencies:update`. after this, all bolt component in `example-integrations` should be up to date. To check if it is working on release we must wait for next release because i don't know how to test it different ten running release.
